pyproject.toml

```diff
@@ -25,9 +25,12 @@ dependencies = [
     "h5py>=3.6.0",
     "igor2",
     "xarray>=0.20.2",
-    "pint==0.17",
     "pynxtools>=0.3.2",
-    "setuptools>=64.0.1"
+    "setuptools>=64.0.1",
+    "pint==0.17; python_version <= '3.9'",
+    "numpy<2.0.0; python_version <= '3.9'",
```

Comment:
```suggestion
    "pint>=0.17,<0.24; python_version <= '3.9'",
    "numpy<2.0.0; python_version <= '3.9'",
```

Reply:
My original idea was to fix it to `pint==0.17` as this is also fixed in NOMAD. But I think for my use case your suggestion is fine as well.

Reply:
@domna it's nice to be explicit with pint<0.24. But in practice this change you suggest won't have an effect, right?  There is no pint==0.24 for py3.9. Or you're covering up some odd case? 

In anycase, I'm quite neutral to this.

Reply:
True, but I just wanted to allow versions in between `0.17` and `0.24` (in case of security fixes etc.). If we enforce `0.17` it might be incompatible with another tool which is installed in the same env. I don't know if we even need `>0.17` or if it can also be lower.